README.md

```diff
@@ -13,11 +13,37 @@ The module is now available on npm! `npm i whatsapp-web.js`
 
 Please note that Node v14+ is required.
 
+## Requirements
```

Comment:
Don't get me wrong. I really enjoy the pro-docker and puppeteer-core changes here. I also believe we should move toward puppeteer-core in the future.

However at this point and since puppeteer is a dependency, I think the Requirements should clearly be listed as optional. And moved closer to the end of the setup and example.

I would structure it this way, following the order in complexity

- Normal Usage
  - Example Normal
- Websocket
  - Changes | only add the changes necessary not the full example
- Docker
  - Setup Docker + Changes | only add the changes necessary not the full example

Extra (if you want to):
You can add the browserWSEndpoint option as a comments in the example.js file.

Reply:
It's done as I got your requests but let me keep clear that still prefer the initial aproach directly. It's more simple and straight to the point.